internal/locate/region_request.go

```diff
@@ -888,6 +902,22 @@ func (s *replicaSelector) updateLeader(leader *metapb.Peer) {
 	s.region.invalidate(StoreNotFound)
 }
 
+// For some reason, the leader is unreachable by now, try followers instead.
+func (s *replicaSelector) fallback2Follower(ctx *RPCContext) bool {
```

Comment:
By now is the only situation that would be used `the stale read fallback -> leader -> fallback replicas`?

Reply:
Yes, when fallbacking to replica from leader, it's a follower read request, not stale read.